src/scales.jl

```diff
@@ -130,7 +130,7 @@ Determine whether `v` should be treated as a continuous or categorical vector.
 """
 function iscontinuous(u)
     v = Broadcast.broadcastable(u)
-    return !haszerodims(v) && eltype(v) <: Union{Number, Date, DateTime}
+    return !haszerodims(v) && eltype(v) <: Union{Number, Date, DateTime, Makie.StaticVector} && !(eltype(v) <: Bool)
```

Comment:
It may be better to set the `Makie.StaticVector` case as `isgeometry`, and then also exclude geometries in the `separate` function (when splitting categorical variables from the rest).